packages/lab/src/combo-box-next/useComboboxPortal.ts

```diff
@@ -20,11 +20,17 @@ export interface UseComboBoxPortalProps {
 
 export function useComboboxPortal(props?: UseComboBoxPortalProps) {
   const {
-    open: openProp = false,
+    open: openProp,
     onOpenChange: onOpenChangeProp,
     placement: placementProp = "bottom",
   } = props || {};
-  const [open, setOpen] = useState<boolean>(openProp);
+
+  const [open, setOpen] = useControlled({
+    controlled: openProp,
+    default: openProp,
```

Comment:
`default` should not share value with `controlled`, or change of `open` would give console warning from `useControlled`